egs/grid/VSR/install.sh

```
@@ -0,0 +1,55 @@
+
+
+
+cd $ICEFALL_DIR/egs/grid/VSR
+if [ $stage -le 3 ] ; then
+  pip install "pip<24.1"
+  pip install -r grid-requirements.txt
+  conda install -y -c conda-forge dlib==19.18.0
+ 
+  pip install  --force-reinstall --no-deps numpy==1.23.5
+
+  echo 'export PYTHONPATH=/data/git/icefall:$PYTHONPATH'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)